test_data/posenet_mobilenet_v1_075_353_481_quant_decoder_edgetpu_reference.csv

```diff
@@ -0,0 +1,35 @@
+pose_id,pose_score,keypoint_label,keypoint_score,keypoint_x,keypoint_y
+0,0.8896713,KeypointType.NOSE,0.9838372,203.62941,97.840675
+0,0.8896713,KeypointType.LEFT_EYE,0.74306124,204.86388,93.445206

```

Comment:
right so this would suggest KeyPointType is still needed if you're comparing against this csv.

Reply:
I'll rerun the generation with just the enum name